xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.h

```diff
@@ -50,6 +50,8 @@ class CRenderManager;
 class IRenderMsg
 {
   friend CRenderManager;
+public:
+  virtual ~IRenderMsg() = default;
```

Comment:
I did, on macOS, using the respective clang warning. So, only non-Mac classes still need to be checked.